nextjs-13/src/components/AppLayout/index.test.tsx

```diff
@@ -1,15 +1,67 @@
 import { render, screen } from '@testing-library/react';
+import { redirect, usePathname } from 'next/navigation';
+import { useSession } from 'next-auth/react';
 
 import AppLayout from './';
 
+jest.mock('next-auth/react');
+jest.mock('next/navigation');
+
 describe('AppLayout', () => {
-  it('renders the component', () => {
-    render(
-      <AppLayout>
-        <></>
-      </AppLayout>
-    );
-
-    expect(screen.getByTestId('layout-default')).toBeVisible();
+  beforeEach(() => {
+    usePathname.mockReturnValue('localhost:3003/send');
```

Comment:
Why is this required?

Reply:
Because I'm using `usePathname ` [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/files#diff-45d2814aa9e2f6cef0f484587e90fdb3161fcc6026a7816e1e4df7fdbd870ea9R11-R12) to get the path name to [differentiate the background-color of the selected tabs and non-selected tabs](https://github.com/nimblehq/js-framework-benchmark/pull/123/files#diff-45d2814aa9e2f6cef0f484587e90fdb3161fcc6026a7816e1e4df7fdbd870ea9R27) 👀 

<img width="278" alt="CleanShot 2023-06-21 at 13 14 45@2x" src="https://github.com/nimblehq/js-framework-benchmark/assets/76901736/ad9c7351-5db9-4303-802b-82e26f9de227">

<img width="283" alt="CleanShot 2023-06-21 at 13 15 21@2x" src="https://github.com/nimblehq/js-framework-benchmark/assets/76901736/39b4349c-6c26-41e8-b7fa-9e5d9ffcfc3d">

Reply:
So you should add a comment above the code as it is quite unexpected

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/commits/6a509538995f3fa7b0bcd282e2f6ebcbb6e6466e) 👀